pkg/reconciler/instances/connectivityproxy/command_test.go

```diff
@@ -148,6 +148,7 @@ func TestCommands(t *testing.T) {
 		kubeClient.AssertExpectations(t)
 	})
 
```

Comment:
Its more like checking if installation for empty manifest will not produce any error. Maybe let's leave it for now. Maybe it will be useful during refactoring

Reply:
Resolved - the code will stay for now.